fix(learn/css): elaborate more about universal selector (#34321)
mdn/content: files/en-us/learn/css/building_blocks/selectors/type_class_and_id_selectors/index.md

## The universal selector

> The universal selector is indicated by an asterisk (`*`). It selects everything in the document. Or it selects everything inside a parent element if `*` is being chained using a [descendant combinator](/en-US/docs/Web/CSS/Descendant_combinator), for example `p *` selects all the elements under the `<p>` element. In the following example, we use the universal selector to remove the margins on all elements. Instead of the default styling added by the browser — which spaces out headings and paragraphs with margins — everything is close together.

Reviewer: ```suggestion
The universal selector is indicated by an asterisk (`*`). It selects everything in the document. If `*` is chained using a [descendant combinator](/en-US/docs/Web/CSS/Descendant_combinator), it selects everything inside that ancestor element. For example, `p *` selects all the nested in the `<p>` element.

In the following example, we use the universal selector to remove the margins on all elements. Instead of the browser's default styling, which spaces out headings and paragraphs with margins, everything is close together.
```

Author: ```suggestion
The universal selector is indicated by an asterisk (`*`). It selects everything in the document. If `*` is chained using a [descendant combinator](/en-US/docs/Web/CSS/Descendant_combinator), it selects everything inside that ancestor element. For example, `p *` selects all the nested elements in the `<p>` element.

In the following example, we use the universal selector to remove the margins on all elements. Instead of the browser's default styling, which spaces out headings and paragraphs with margins, everything is close together.
```
